CHANGELOG.md

```diff
@@ -25,10 +16,37 @@
     - **aviatrix_spoke_gateway**
     - **aviatrix_edge_spoke_gateway**
     - **aviatrix_transit_gateway**
-5. Added new attribute ``bgp_bfd`` and ``enable_bfd`` to support bgp_bfd configuration in the following resources
+2. Added new attribute ``bgp_bfd`` and ``enable_bfd`` to support bgp_bfd configuration in the following resources.
     - **aviatrix_transit_external_device_conn**
     - **aviatrix_edge_spoke_external_device_conn**
     - **aviatrix_spoke_external_device_conn**
+3. Added/Updated Edge Terraform documentation to include interface mapping.
+4. Updated documentation for ``aviatrix_firewall_install`` where `user_data` was not applicable to Palo Alto firewalls. There is no longer
```

Comment:
```suggestion
4. Updated documentation for **aviatrix_firewall_instance** where ``user_data`` was not applicable to Palo Alto firewalls. There is no longer
```

In order to follow current syntax, bolded for resource and "``" quotations to be used for attributes. Also is a typo for the resource type